tico/serialize/circle_graph.py

```diff
@@ -208,6 +200,7 @@ def add_tensor_from_scratch(
         self,
         prefix: str,
         shape: List[int],
+        shape_signature: Optional[List[int]],
         dtype: int,
```

Comment:
```suggestion
        dtype: int,
        shape_signature: Optional[List[int]] = None,
```

Then, the change of `test/unit_test/serialize_test/test_circle_graph.py` wouldn't be necessary.

Reply:
Changing some unit tests wasn't big. 
I think it's better to manually passing None to shape_signature.